include/openssl/ssl.h

```diff
@@ -217,11 +217,9 @@ typedef struct srtp_protection_profile_st {
 
 DEFINE_STACK_OF(SRTP_PROTECTION_PROFILE)
 
-typedef int (*tls_session_ticket_ext_cb_fn) (SSL *s,
-                                             const unsigned char *data,
+typedef int (*tls_session_ticket_ext_cb_fn) (SSL *s, const unsigned char *data,
```

Comment:
A style pass over ssl.h might reasonably be expected to normalize on one of ') (' or ')(' in function typedefs.
https://github.com/openssl/openssl/pull/2279#discussion_r102722095 is perhaps in support of the latter (no space), but we've really been all over the place.

Reply:
I would remove the space.

Reply:
fixed